server/front/src/starter.ts

```diff
@@ -116,6 +116,19 @@ export function startFront (extraConfig?: Record<string, string>): void {
     process.exit(1)
   }
 
+  const uploadMaxFileSize = Number(process.env.MAX_FILE_SIZE)
```

Comment:
These changes won't work without setting env variables. Once we merge the pull request, the file upload will be broken. We should either set env variables, or do not apply limits if variables are not set.